python.md

```diff
@@ -125,22 +125,41 @@ to the number of threads to use, the default is to use a single thread.
 
 ## 3. Other common Python packages
 
+### 3.1 cffi
```

Comment:
@geoffreyblake should we make cffi as part of the prerequisite for all this

Reply:
Updated this to include libffi-dev/libffi-devel in the general requirements at the top of the document.